### ISSUE TITLE
68 to 468?

### ISSUE THREAD

**Issue by ghost** [PeizhiYan/Mediapipe_2_Dlib_Landmarks#1]
Can you add the ability to convert 68 to 468?

**Comment by PeizhiYan** [comment]
Hi! You may want to try this  https://pypi.org/project/dlib2mediapipe/ 
But I haven't tested it yet.

**Comment by ghost** [comment]
The code isn't there

**Comment by PeizhiYan** [comment]
Yes, that link I shared is a package and I did not find their source code either. 

A straightforward solution for converting dlib landmarks to mediapipe landmarks is by fitting a 3DMM model (FLAME) using the dlib landmarks only, then directly get the Mediapipe landmarks from the FLAME mesh. There are landmark correspondences you can find online. It is doable but not the focus of this repo.
